### PR TITLE
feat: support brotli

### PR DESCRIPTION
**Pull request by clarkdo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ The following compression codings are supported:
 
   - deflate
   - gzip
+  - brotli
```
Comment (dougwilson) [inline README.md]: The coding name for Brotli is `br`, which is what should be listed here.

**Comment by clarkdo** [comment]
@dougwilson Thank you so much for the quick and detailed review.

For the `options`, which solution do you think is better ?

1. Try to match current options into brotli options for keeping consistency
2. Add a new brotli options field in options of compression

For the test, since brotli is only supported in node 11.7, can we add it in travis ?

**Comment by manniL** [comment]
Friendly reminder @dougwilson ☺️

File: index.js [hunk 2]
```diff
@@ -190,9 +199,11 @@ function compression (options) {
 
       // compression stream
       debug('%s compression', method)
-      stream = method === 'gzip'
-        ? zlib.createGzip(opts)
-        : zlib.createDeflate(opts)
+      switch (method) {
+        case 'br': stream = zlib.createBrotliDecompress(opts); break
```
Comment (cecchi) [inline index.js]: I believe this should be `createBrotliCompress`, not `createBrotliDecompress`, right?

Comment (clarkdo) [reply index.js]: Thanks 😄

**Comment by svewag** [comment]
Hi there 👋 

I would really like to serve my static files with brotli. So are there any plans when this might be merged or can I help out somewhere?

Greetings Sven

**Comment by Saiv46** [comment]
Any progress?

**Comment by jeremytenjo** [comment]
Any progress?

**Comment by HamzaAnis** [comment]
This PR is long due. If some one can review it then that would be great.

**Comment by Saiv46** [comment]
Unsubscribed, looks like this will be never merged.

Workaround:

Set up nginx proxy in front of expressjs server, as bonus you can also do TLS termination and round-robin balancing.

**Comment by bjohansebas** [comment]
Closed in favor of #194, @clarkdo thank you for the initiative. You can continue helping by reviewing #194 so we can launch it.

**Pull request closed without merging**